src/components/application_manager/include/application_manager/app_service_manager.h

```diff
@@ -164,6 +163,12 @@ class AppServiceManager {
       const smart_objects::SmartObject& service_record,
       const mobile_apis::ServiceUpdateReason::eType update_reason,
       smart_objects::SmartObject& msg_params);
+  void GetProviderFromService(const AppService& service,
+                              ApplicationSharedPtr& app,
+                              bool& hmi_service);
+  std::pair<std::string, AppService> FindServiceByAppID(std::string name,
+                                                        std::string type);
+  std::string GetServiceAppID(AppService service);
```

Comment:
I would specify it as PolicyAppID since `AppID` means something different internally to core

Reply:
Same with FindServiceByAppID